spec/controllers/join_requests_controller_spec.rb

```diff
@@ -344,6 +345,34 @@
       it { should redirect_to(invite_space_join_requests_path(space)) }
       it { should set_the_flash.to(I18n.t('join_requests.create.error', :errors => errors)) }
     end
+
+    context "global admin is not a member of the space and successfully invites one user" do
```

Comment:
I think we should start referencing the tickets/PRs on tests made specifically for them. Like: `"global admin is not a member of the space and successfully invites one user (RM-1455)"`.

* `RM-xxxx` for issues on redmine
* `PR-xx` for pull requests
* Something else?

What do you think?